pkg/adapter/kubernetesevents/cloudevent_test.go

```diff
@@ -55,10 +55,11 @@ func TestCloudEventFrom(t *testing.T) {
 
 	want := cloudevents.Event{
 		Context: cloudevents.EventContextV02{
-			Type:   eventType,
-			ID:     string(uid),
-			Source: *cetypes.ParseURLRef(refLink),
-			Time:   &cetypes.Timestamp{Time: now},
+			SpecVersion: "0.2",
```

Comment:
No need for action, just a comment that I find it counterintuitive that by using the EventContextV02 you still have to specify SpecVersion as 0.2.

Reply:
This is on the Want side. you don't if you ask for it as for the struct back `.AsV02()` it fill out that field for you.

Reply:
duh, thanks :)